cwms-servers/CWMS_HOME/bin/loggingSetup.py

```diff
@@ -26,6 +24,7 @@ def setup_logging():
         (logging.getLogger("logging_setup")
          .warning("Could not load python logging config. CWMS_HOME not defined."))
     else:
+        sys.path.append("{}/bin/lib".format(cwms_home))
```

Comment:
same here, do you want to print the path out?

Reply:
updated